# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/mnemoc/clip/no_clean.patch
# ROCK Linux is Copyright (C) 1998 - 2004 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./cliplibs/Makefile.orig	2004-01-08 19:01:54.000000000 -0300
+++ ./cliplibs/Makefile	2004-01-08 19:22:08.000000000 -0300
@@ -5,7 +5,7 @@
        clip-mysql clip-postgres clip-oracle clip-interbase clip-odbc \
        clip-com clip-oasis clip-rtf \
 
-all local user system install lib: clean
+all local user system install lib: 
 	for dir in $(dirs); \
 	do \
 		(cd $$dir ;\
